src/Avalonia.Visuals/Animation/RenderLoopClock.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Avalonia.Rendering;
+
+namespace Avalonia.Animation
+{
+    public class RenderLoopClock : ClockBase, IRenderLoopTask
+    {
+        protected override void Stop()
+        {
+            AvaloniaLocator.Current.GetService<IRenderLoop>().Remove(this);
+        }
+
+        bool IRenderLoopTask.NeedsUpdate => HasSubscriptions;
+
+        void IRenderLoopTask.Render()
```

Comment:
Hmm, `Render` might not be the best name for this method? A clock doesn't "render". `Pulse` maybe? `Execute`?

Reply:
I think `Render` works better for the `IRenderLoopTask` interface. And since this method is unused in `RenderLoopClock` anyway I would say it's fine. I can change it though. Would make the naming more generalized if other people want to add on their own non-rendering tasks to the render loop.

Reply:
No, leave it as it is, no particularly strong feelings on this.